coroutines-codelab/finished_code/src/test/java/com/example/android/kotlincoroutines/main/utils/LiveDataTestExtensions.kt

```diff
@@ -34,30 +31,12 @@ class LiveDataValueCapture<T> {
     val values: List<T?>
         get() = _values
 
-    val channel = Channel<T?>(Channel.UNLIMITED)
+    private val channel = Channel<T?>(Channel.UNLIMITED)
```

Comment:
Same comment about Channel as above

Reply:
+1

Reply:
Even better, dead code that can be removed 🎉